azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/model/DeployType.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.toolkits.appservice.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Getter
+@AllArgsConstructor
+public class DeployType {
+    public static final DeployType WAR = new DeployType("war");
+    public static final DeployType JAR = new DeployType("jar");
+    public static final DeployType EAR = new DeployType("ear");
+    public static final DeployType JAR_LIB = new DeployType("lib");
+    public static final DeployType STATIC = new DeployType("static");
+    public static final DeployType SCRIPT_STARTUP = new DeployType("startup");
+    public static final DeployType ZIP = new DeployType("zip");
+
+    private String value;
+
+    public List<DeployType> values() {
```

Comment:
enum?

Reply:
We may discuss whether to use enum or Constant later and refactor in single pr